src/avro/src/types.rs

```diff
@@ -99,7 +97,7 @@ impl From<Scalar> for Value {
 // This simple wrapper exists so we can Debug-print the values deterministically, i.e. in sorted order
 // by keys.
 #[derive(Clone, PartialEq)]
-pub struct AvroMap(pub HashMap<String, Value>);
+pub struct AvroMap(pub BTreeMap<String, Value>);
```

Comment:
We can probably get rid of this type, since it only exists to make its `Debug` implementation print things in sorted order, and change `Value::Map(AvroMap)` below to `Value::Map(BTreeMap<String, Value>)`.

Reply:
Could catch! I've added a second commit to remove it. Can you take another look?